libs/coin-modules/coin-evm/src/__tests__/coin-tester/anvil.ts

```diff
@@ -0,0 +1,55 @@
+import { v2 as compose } from "docker-compose";
```

Comment:
If we still want to upgrade to docker-compose v3, that should probably be changed as well.

Reply:
Good point. The library doesn't support v3 yet should we probably stick to `2.4` actually